src/pages/privacy-statement.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import { Translation } from 'react-i18next';
+import { withI18next } from '@wapps/gatsby-plugin-i18next';
+import { suomifiDesignTokens } from 'suomifi-ui-components';
+
+import Layout from 'components/layout';
+import SEO from 'components/seo';
+import { Heading } from 'components/ResponsiveComponents';
+import Section, { Props as SectionProps } from 'components/Section';
+
+const Page = (): JSX.Element => (
+  <Translation ns={['privacy-statement']}>
+    {(t) => (
+      <Layout>
+        <div style={{ padding: `0 ${suomifiDesignTokens.spacing.xl}` }}>
```

Comment:
I had already forgotten we are using inline styles here. 😓  This whole app needs to be rewritten for so many reasons.

Reply:
Yes. Hopefully we have the time to do that at some point.